types/markdown-it/lib/rules_block/state_block.d.mts

```diff
@@ -80,18 +80,12 @@ export default class StateBlock {
     listIndent: number;
 
     /**
-     * can be 'blockquote', 'list', 'root', 'paragraph' or 'reference'
      * used in lists to determine if they interrupt a paragraph
      */
     parentType: ParentType;
 
     level: number;
 
-    /**
-     * renderer
-     */
-    result: string;
```

Comment:
this is not mentioned anywhere in https://unpkg.com/browse/markdown-it@14.1.0/lib/rules_block/state_block.mjs